packages/clerk-js/src/core/devBrowserHandler.ts

```diff
@@ -38,11 +38,17 @@ export default function createDevBrowserHandler({
 
   function setDevBrowserJWT(jwt: string) {
     localStorage.setItem(key, jwt);
+    // Append dev browser JWT to cookies, because Next.js middleware
```

Comment:
nitpick: this is not specific to nextjs, let's update the comment so it doesn't confuses us in the future.